includes/payment-methods/class-wc-stripe-upe-payment-method-link.php

```diff
@@ -87,10 +87,15 @@ public function is_available_for_account_country() {
 	/**
 	 * Returns true if the UPE method is available.
 	 *
+	 * Link isn't like a traditional UPE payment method as it is not shown as a standard payment method at checkout.
+	 * Customers use the Stripe Link button and the existing credit card fields to enter their payment details. The payment is then treated as a card.
+	 *
+	 * We return false here so the payment method isn't considered available by WooCommerce and rendered as a payment method at checkout.
+	 *
 	 * @return bool
 	 */
 	public function is_available() {
-		return $this->is_available_for_account_country() && parent::is_available();
+		return false;
```

Comment:
On it's own, this change looks a little scary because changing a function to simply return `false`, however, after considering the history of this function, I I think this change is fine. 

On `develop` the `is_available()` function in `WC_Stripe_UPE_Payment_Method` use to return `true`. [[code ref](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/includes/payment-methods/class-wc-stripe-upe-payment-method.php#L122-L124)]. 

The UPE link method had a bit more logic in it's `is_available()` function [[code link](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/includes/payment-methods/class-wc-stripe-upe-payment-method-link.php#L79-L85)], however, in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2820 it was changed because `is_available()` is an inherited function of `WC_Payment_Gateway` and so when we changed all UPE elements to be `WC_Payment_Gateway`s, these `is_available()` functions took on a new meaning as WooCommerce uses this `is_available()` function to determine which payment methods to display to customers on the checkout. 

With this in mind, I think returning false because we don't want WooCommerce to display this method on the checkout makes logical sense. Let me know if you see an issue with this approach. 

In my testing, making purchases with Link still works fine.